nixos/modules/services/home-automation/openthread-border-router/openthread-border-router-setup.exp

```diff
@@ -0,0 +1,82 @@
+proc expect_line {line} {
```

Comment:
if you wanted to make this a bash script, `ot-ctl` can also take commands as command line arguments - e.g. `ot-ctl dataset init new` without needing to read back